__tests__/app.test.js

```diff
@@ -234,11 +234,87 @@ describe.only("POST /api/articles/:article_id/comments", () => {
       body: "Loved this article!",
     };
     return request(app)
-      .post("/api/articles/invalid_id/comments")
+      .post("/api/articles/999999/comments")
       .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+  test("404:POST/api/articles/:article_id/comments returns 404 when given a username that doesnt exist", () => {
+    const newComment = {
+      username: "i_dont_exist",
+      body: "Loved this article!",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("User not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200:PATCH /api/articles/:article_id returns an updated article when incrementing votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: 1,
+          votes: 101,
+        });
+      });
+  });
```

Comment:
This is clean because you're only checking for the _article_id_ and _votes_ properties, however, as the ticket says 
```
Responds with the updated article
```

It would probably be better to test more robustly here and check that the whole article object is present in the response